packages/core/components/Puck/components/ResizeHandle/index.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useCallback, useEffect, useRef } from "react";
+import getClassNameFactory from "../../../../lib/get-class-name-factory";
+import styles from "./styles.module.css";
+import { useCanvasFrame } from "../../../../lib/frame-context";
+
+const getClassName = getClassNameFactory("ResizeHandle", styles);
+
+interface ResizeHandleProps {
+  position: "left" | "right";
+  sidebarRef: { current: HTMLDivElement | null };
+  onResize: (width: number) => void;
+  onResizeEnd: (width: number) => void;
+}
+
+export const ResizeHandle: React.FC<ResizeHandleProps> = ({
+  position,
+  sidebarRef,
+  onResize,
+  onResizeEnd,
+}) => {
+  const { resetAutoZoom } = useCanvasFrame();
+
+  const handleRef = useRef<HTMLDivElement>(null);
+  const isDragging = useRef(false);
+  const startX = useRef(0);
+  const startWidth = useRef(0);
+
+  const handleMouseMove = useCallback(
+    (e: MouseEvent) => {
+      if (!isDragging.current) return;
+
+      const delta = e.clientX - startX.current;
+      const newWidth =
+        position === "left"
+          ? startWidth.current + delta
+          : startWidth.current - delta;
+
+      const width = Math.max(186, newWidth);
```

Comment:
Puck uses multiples of 4

```suggestion
      const width = Math.max(192, newWidth);
```

Reply:
Updated ✅
I noticed some old values inside packages/core/components/Puck/styles.module.css (rows 51 and 81) that might also need an update